.gitignore

```diff
@@ -66,3 +66,8 @@ build-iPhoneSimulator/
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+# Aux
+organizations_controller copy.rb
+organization_serializer copy.rb
+organizations_serializer copy.rb
+paginable copy.rb
```

Comment:
Por favor 🙏 revisar la configuración de EOF en tu IDE, te paso la solución para [VS Code](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files).